tests/fixtures/ecma-version/9/regexp-lookbehind-assertion/valid-negative-1.result.js

```diff
@@ -1,3 +1,5 @@
+const condtionalRegex = require("../../../../lib/conditional-regex-value");
```

Comment:
Nitpick: I think the variable name is misspelled here (it is missing an "i" after "cond").